src/ontology/cl-edit.owl

```diff
@@ -32517,6 +32519,18 @@ AnnotationAssertion(rdfs:label obo:CL_4042027 "GABAergic interneuron of the post
 SubClassOf(obo:CL_4042027 obo:CL_0002614)
 SubClassOf(obo:CL_4042027 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001903))
 
+# Class: obo:CL_4042032 (PAX6 GABAergic cortical interneuron)
+
+AnnotationAssertion(obo:IAO_0000115 obo:CL_4042032 "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses PAX6. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters PAX6.")
+AnnotationAssertion(terms:contributor obo:CL_4042032 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042032 "2024-10-11T13:25:51Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4042032 "PMID:37824655")
```

Comment:
Is this reference meant to annotate the definition?

Reply:
It is meant to annotate the definition by looking at the scRNAseq data